fish-pepper/run-java-sh/fp-files/java-default-options

```diff
@@ -78,8 +78,8 @@ calc_mem_opt() {
   local max_mem="$1"
   local fraction="$2"
   local mem_opt="$3"
-  
-  local val=$(echo "${max_mem} ${fraction} 1048576" | awk '{printf "%d\n" , ($1*$2)/(100*$3) + 0.5}')
+
+  local val=$(calc "${max_mem}" "${fraction}" "ceil(\$1*\$2/100/1048576)" "%d\n")
```

Comment:
I think that's slightly different because the original algorithm is not about getting the ceiling but about rounding (hence that + 0.5). Maybe we can add a `round` to the functions, too?

BTW, integration tests for the java opts calculations are still missing, so not sure whether it provides the correct results yet. Planning to add some tests later this evening.

But if you like you could give a try. Just add a `java_options.bats` to `tests/t` and call top-level `run_all.sh` (see also top-level README)

Reply:
Ah good catch. Let me add the `round` function too and see if I can add some tests!